client/source/class/sar/steps/AnalysisCreation.js

```diff
@@ -131,6 +133,8 @@ qx.Class.define("sar.steps.AnalysisCreation", {
         column: 0,
         colSpan: 2
       });
+      this.__xArea = modelEditor._form.getItem("measAreaX");
```

Comment:
could you please call them `modelAreaX` and `modelAreaY` to avoid (my own) confusion with the test set generation?

Reply:
thank you!